pyproject.toml

```diff
@@ -25,7 +25,7 @@ bambi = "^0.13.0"
 numpyro = "^0.15.0"
 hddm-wfpt = "^0.1.4"
 seaborn = "^0.13.2"
-jax = "^0.4.23"
+jax = { version = "^0.4.23", extras = ["cuda12"], optional = true }
```

Comment:
I think for now we have to cap jax roughly at <= 0.4.26, because of the config issue... setting floatX breaks on newest.

Reply:
Looks like they fixed this? I have jax==0.4.28 and this issue did not pop up for me...

Reply:
Fixed